limitador/src/lib.rs

```diff
@@ -226,6 +226,49 @@ pub struct CheckResult {
     pub limit_name: Option<String>,
 }
 
+impl CheckResult {
+    pub fn response_header(&mut self) -> HashMap<String, String> {
```

Comment:
Thinking this should return something other that `String` as keys to our `Map` here, but not a big deal (I think) for now as there is only one supported impl.

Reply:
Thanks for refactoring this here.